FileManager/Models/FilesUnit.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  FilesUnit.swift
+//  FileManager
+//
+//  Created by Екатерина Неделько on 5.06.22.
+//
+
+import Foundation
+
+struct FilesUnit: Equatable {
+    let name: String
+    let path: URL
+    
+    let type: FilesUnitType
+}
+
+enum FilesUnitType {
+    case folder
+    case image
+    case photo
```

Comment:
What would be the difference between `image` and `photo`?

Reply:
image - upload with PHPicker, photo - upload with ImagePicker 
I use handleNewFileButtonTap method in which, depending on the type, I create an image (using PHPicker) or a photo (using ImagePicker)